pkg/ephemeral/network/proxy_test.go

```diff
@@ -31,10 +31,12 @@ var _ = Describe("Proxy", func() {
 				Act: &Activation{
 					GameID: "a",
 				},
-				Proxy: &ProxyConfig{
-					Host:      "localhost",
-					Port:      "5001",
-					LocalPort: "5000",
+				ProxyEntries: []*ProxyConfig{
+					{
+						Host:      "localhost",
+						Port:      "5001",
+						LocalPort: "5000", //ToDo: Switch to ports that are NOT used by everyone else
```

Comment:
I had an issue with this when running the test locally as I had port 5000 blocked.
So that proxy would try to get the port,  block and the test would never complete.